build/helper/codegen_helper.py

```diff
@@ -251,8 +253,8 @@ def _get_ctype_variable_definition_snippet_for_scalar(parameter, parameters, ivi
         S160. Input is size of input buffer:                                       visatype.ViInt32(0 if list is None else len(list))
         S170. Input is size of output buffer with mechanism ivi-dance, QUERY_SIZE: visatype.ViInt32()
         S180. Input is size of output buffer with mechanism ivi-dance, GET_DATA:   visatype.ViInt32(error_code)
-        S190. Input is size of output buffer with mechanism passed-in:             visatype.ViInt32(buffer_size)
-        S200. Output scalar or enum:                                               visatype.ViInt32()
+        S210. Input is size of output buffer with mechanism passed-in:             visatype.ViInt32(buffer_size)
```

Comment:
Looks like S190 is still used but it's gone from documentation?
I really think that adding `ivi-dance-with-a-twist` support (with NI-FAKE coverage) should be its own PR separate from adding nidigital.